impeller/entity/contents/solid_stroke_contents.cc

```diff
@@ -95,8 +91,17 @@ static VertexBuffer CreateSolidStrokeVertices(
     std::tie(contour_start_point_i, contour_end_point_i) =
         polyline.GetContourPointBounds(contour_i);
 
-    if (contour_end_point_i - contour_start_point_i < 2) {
-      continue;  // This contour has no renderable content.
+    switch (contour_end_point_i - contour_start_point_i) {
+      case 1: {
+        Point p = polyline.points[contour_start_point_i];
```

Comment:
Nice catch that this index should be `contour_start_point_i`.

Reply:
Oh, haha, I didn't cut/paste your code, I just went "aha" and wrote it manually. So I didn't even really catch anything...